test/caption-stream-test.js

```diff
@@ -525,44 +571,107 @@
+      },
+      // backspace
+      { ccData: 0x1421 },
+      // backspace
+      { ccData: 0x1421 }, // duplicate is ignored
+      // backspace
+      { ccData: 0x1421 },
+      // CR, carriage return
+      { pts: 1 * 1000, ccData: 0x142d }
+    ].forEach(cea608Stream.push, cea608Stream);
 
     equal(captions.length, 0, 'no caption emitted');
   });
 
-  test('backspaces stop at the beginning of the line', function() {
+  test('a second identical control code immediately following the first is ignored', function() {
     var captions = [];
     cea608Stream.on('data', function(caption) {
       captions.push(caption);
     });
 
-    // RU2, roll-up captions 2 rows
-    cea608Stream.push({ ccData: 0x1425 });
-    // '01'
-    cea608Stream.push({
-      pts: 0 * 1000,
-      ccData: characters('01')
+    [ // RU2, roll-up captions 2 rows
+      { ccData: 0x1425 },
+      // '01'
+      {
+        pts: 0 * 1000,
+        ccData: characters('01')
+      },
+      // '02'
+      {
+        pts: 1 * 1000,
+        ccData: characters('02')
+      },
+      // backspace
+      { ccData: 0x1421 },
+      // backspace
+      { ccData: 0x1421 }, // duplicate is ignored
+      // backspace
+      { ccData: 0x1421 },
+      // CR, carriage return
+      { pts: 2 * 1000, ccData: 0x142d }
+    ].forEach(cea608Stream.push, cea608Stream);
+
+    equal(captions.length, 1, 'caption emitted');
+    equal(captions[0].text, '01', 'only two backspaces processed');
+  });
+
+  test('preable address codes are converted into spaces', function() {
+    var captions = [];
+    cea608Stream.on('data', function(caption) {
+      captions.push(caption);
     });
-    // backspace
-    cea608Stream.push({ ccData: 0x1421 });
-    // backspace
-    cea608Stream.push({ ccData: 0x1421 });
-    // backspace
-    cea608Stream.push({ ccData: 0x1421 });
-    // CR, carriage return
-    cea608Stream.push({ pts: 1 * 1000, ccData: 0x142d });
+
+    [ // RU2, roll-up captions 2 rows
+      { ccData: 0x1425 },
+      // '01'
+      {
+        pts: 0 * 1000,
+        ccData: characters('01')
+      },
+      // PAC: row 15, indent 0
+      { ccData: 0x1470 },
+      // '02'
+      {
+        pts: 1 * 1000,
+        ccData: characters('02')
+      },
+      // CR, carriage return
+      { pts: 2 * 1000, ccData: 0x142d }
+    ].forEach(cea608Stream.push, cea608Stream);
+
+    equal(captions.length, 1, 'caption emitted');
+    equal(captions[0].text, '01  02', 'PACs are was converted to space');
+  });
+  test('backspaces stop at the beginning of the line', function() {
+    var captions = [];
+    cea608Stream.on('data', function(caption) {
+      captions.push(caption);
+    });
+
+    [ // RU2, roll-up captions 2 rows
+      { ccData: 0x1425 },
+      // '01'
+      {
+        pts: 0 * 1000,
+        ccData: characters('01')
+      },
+      // backspace
+      { ccData: 0x1421 },
+      // backspace
+      { ccData: 0x1421 },
+      // backspace
+      { ccData: 0x1421 },
```

Comment:
The duplicate control code filtering here would make this only two backspaces, right?